beetsplug/bandcamp/_search.py

```diff
@@ -30,6 +30,8 @@ def _f(field: str) -> str:
     re.compile(r"(?P<url>https://bandcamp\.(?P<label>[^.]+)\.(?!bcbits)[\w/.-]+)"),
     # label-second pattern
     re.compile(r"(?P<url>https://(?P<label>[^.]+)\.bandcamp(?!bcbits)[\w/.-]+)"),
+    # alternative domain name pattern
+    re.compile(r"(?P<url>https://(?P<domain>[^\?\"]+)(?!bcbits)/album/[^\?\"]+)\?from=search"),
```

Comment:
That's good stuff, big fan of the fact that you've included `from=search` at the end to ensure it doesn't pick up the rubbish. 

A tiny nit pick: 
```suggestion
    re.compile(r'(?P<url>https://(?P<domain>[^\?"]+)(?!bcbits)/album/[^\?"]+)\?from=search'),
```

this would remove the need for backslashes before the double quotes.

Reply:
Feel free to replace all url patterns with below

```python
    re.compile(r"(?P<url>https://bandcamp\.(?P<label>[^.]+)\.[^?]+)\?from=search"),
    re.compile(r"(?P<url>https://(?P<label>[^.]+)\.bandcamp\.[^?]+)\?from=search"),
    re.compile(r"(?P<url>https://(?P<label>[^.]+)\.[^?]+)\?from=search"),
```

have tested them against all failures I had and they worked well. No need to exclude `bcbits` if we're hardcoding `from=search`, amazing idea!

Reply:
Hi, I did some adjustments along your remarks, with unit testing

Reply:
Beautiful stuff!